File: CompilationTest/ExplicitTypeTest/results/ExplicitTypeTest.cpp
#include <iostream>
#include <string>

class TestClass {
  public:
    void testMethod() {
        auto op = std::string("x");
        std::cout << op.size() << std::endl;
    }

  private:
};

int main()
{
    try {
        TestClass c;
        c.testMethod();
    } catch(std::exception& err) {
        std::cout << "Exception: " << err.what() << '\n';
    }
    return 0;
}

File: CompilationTest/ExplicitTypeTest/results/ExplicitTypeTest.php
<?php

class TestClass {
    function testMethod() {
        $op = "x";
        print(strlen($op) . "\n");
    }
}

try {
    $c = new TestClass();
    $c->testMethod();
} catch (Exception $err) {
    echo 'Exception: ' . $err->getMessage() . "\n";
}